apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -89,6 +95,8 @@ const handleTicketEvents = async (requestData) => {
                     ticketId: updatedItem.id,
                     ticketNumber: updatedItem.number,
                     userId: nextAssigneeId || prevAssigneeId,
+                    url: `${conf.SERVER_URL}/ticket/${updatedItem.id}`,
```

Comment:
You have two similar parameters for one task: URL and domain. Why do you need both?